server/controllers/homepage.js

```diff
@@ -45,8 +84,8 @@ module.exports = function(app) {
       getTotalCollectives(),
       getTotalDonors(),
       queries.getTotalDonations(),
-      getGroupsByTag('open source'),
-      getGroupsByTag('meetup')
+      getGroupsByTagForCollectiveCard('open source'),
+      getGroupsByTagForCollectiveCard('meetup')
```

Comment:
I did not rename the function, I created another function that extends `getGroupsByTag`, specifically to supply the information that is displayed at the footer of the CollectiveCard component.

https://github.com/OpenCollective/opencollective-api/pull/478/files#diff-9c36a551a062a029a91b82d5299327ebR25